portfolio/src/main/java/com/google/sps/servlets/UpvoteServlet.java

```diff
@@ -0,0 +1,18 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/load-comments")
```

Comment:
I know this can be annoying if you already have plans to use this code later, but it's best to remove any unused or incomplete code from this PR.

As code reviewers, it's better to play it safe and assume that people may never get around to completing follow-up PRs, and so we should only approve the PR based on what's actually in the PR and not what's planned.